src-tauri/src/plugin_management/mod.rs

```diff
@@ -0,0 +1,75 @@
+use app::GameInstall;
+use once_cell::sync::Lazy;
+use std::{
+    fs::{self, File, OpenOptions},
+    io,
+    path::PathBuf,
+};
+use tauri::async_runtime::Mutex;
+use thermite::{core::utils::TempDir, prelude::ThermiteError};
+use zip::ZipArchive;
+
+static CAN_INSTALL_PLUGINS: Lazy<Mutex<bool>> = Lazy::new(|| Mutex::new(false));
```

Comment:
It might make more sense to pass the `can_install_plugins` as an argument to `fc_download_mod_and_install` via frontend cause that way there's no need to keep track of state in the backend.

In general I try to keep state in frontend so that backend can act completely stateless which means that locking variables etc can be completely omitted ^^